src/main/java/org/myrobotlab/opencv/VideoProcessor.java

```diff
@@ -62,7 +63,7 @@
   transient public BlockingQueue<Object> blockingData = new LinkedBlockingQueue<Object>();
   private transient FrameGrabber grabber = null;
   public transient Thread videoThread = null;
-  private transient Map<String, OpenCVFilter> filters = new LinkedHashMap<String, OpenCVFilter>();
+  private transient LinkedHashMap<String, OpenCVFilter> filters = new LinkedHashMap<String, OpenCVFilter>();
   private transient List<OpenCVFilter> addFilterQueue = new ArrayList<OpenCVFilter>();
```

Comment:
Map<String, OpenCVFilter> filters = new LinkedHashMap<String, OpenCVFilter>() 
Using Map is preferred.  Map is an "interface" and LinkedHashMap is the implementation. If the code uses the interface, a different implementation can easily be switched